Podfile.lock

```diff
@@ -1,5 +1,5 @@
 PODS:
-  - MiniApp (2.5.0):
+  - MiniApp (2.6.0):
```

Comment:
Isn't this supposed to be 2.7.0 🤔 ? We have 2.7.0 released in Pods already right?

Reply:
yeah, this is just podlock updated before the release, we don't have to worry about this